### PR TITLE
feat(dlm):adapt to task framework

### PR DESCRIPTION
**Pull request by guowl3**
#### What type of PR is this?

adapt data-archive job to task framework.

#### Which issue(s) this PR fixes:

Fixes #

#### Additional documentation e.g., usage docs, etc.:

```docs

```

### CODE REVIEW

**Review by krihy — DISMISSED** [review]
LGTM

File: server/odc-migrate/src/main/resources/migrate/common/V_4_2_3_16__alter_schedule_task.sql [hunk 1]
```diff
@@ -0,0 +1,2 @@
+alter table `schedule_task` add column `job_id` bigint DEFAULT NULL COMMENT 'job id, references job_job(id)';
```
Comment (yhilmare) [inline server/odc-migrate/src/main/resources/migrate/common/V_4_2_3_16__alter_schedule_task.sql:1]: this version is occupied

**Review by krihy — APPROVED** [review]
LGTM

**Review by krihy — APPROVED** [review]
lgtm

**Pull request merged**